src/index.ts

```diff
@@ -4,6 +4,10 @@ import {createCompiler} from '@mdx-js/mdx'
 import matter from 'gray-matter'
 import {build as bundle, Plugin, BuildOptions} from 'esbuild'
 import nodeResolve from '@esbuild-plugins/node-resolve'
+import {
+  globalExternals,
+  ModuleInfo,
+} from '@fal-works/esbuild-plugin-global-externals'
```

Comment:
I think that I have `isolatedModules` enabled in `tsconfig.json` so I think this will need to change to:

```suggestion
import {globalExternals} from '@fal-works/esbuild-plugin-global-externals'
import type {ModuleInfo} from '@fal-works/esbuild-plugin-global-externals'
```